components/usage-api/usage/v1/billing.proto

```diff
@@ -50,3 +53,14 @@ message FinalizeInvoiceRequest {
 
 message FinalizeInvoiceResponse {
 }
+
+// If there are two billable sessions for this instance ID,
+// the second one's "from" will be the first one's "to"
+message SetBilledSessionRequest {
```

Comment:
Here, we're more less encoding a BilledSession object, just as fields. I think we could do the following:

```
message BilledSession {
	 string instance_id = 1;
  	 google.protobuf.Timestamp from = 2;
  	 string system = 3;
}

message SetBilledSessionRequest {
  BilledSession session = 1;
}

message SetBilledSessionResponse {
  BilledSession session = 1;
}
```

Why? Because we're then re-using a domain object (BilledSession) for the Set (which also returns the object with it's values (and any possibly set by the creation process)

Reply:
I'll leave this for now for when I continue with the implementation and have a better idea of what the response looks like 🙏🏽